README.md

```diff
@@ -0,0 +1,37 @@
+## Sorting tool:
```

Comment:
Usually we use `#` for the project name

`#` - This is a ` <h1>` tag.  
`##`  - This is a  `<h2>` tag